spread.yaml

```diff
@@ -57,6 +57,24 @@ backends:
                 username: ubuntu
                 password: ubuntu
 
+    external-ubuntu-core:
+        type: adhoc
+        environment:
+            SPREAD_ADHOC_UC_ADDRESS: "$(HOST: echo ${SPREAD_ADHOC_UC_ADDRESS:-localhost:8022})"
+        allocate: |
+            ADDRESS $SPREAD_ADHOC_UC_ADDRESS
+        systems:
+            # FIXME: remove the ubuntu-core-16.04-64 dummy system once the runner handles tasks without systems assigned
+            - ubuntu-core-16.04-64:
+                username: test
+                password: ubuntu
+            - ubuntu-core-16-device-amd64:
+                username: test
+                password: ubuntu
+            - ubuntu-core-16-device-arm:
```

Comment:
Yes, let's please duplicate. There's no difference because the actual systems are being externally cooked, so the differences are being encoded elsewhere, but they exist and if we support these systems somewhere else (hopefully we will!) we'll want to run two copies of the tests, one on each of those systems, with different images.